src/app/settings/admin/presets/style.scss

```diff
@@ -68,3 +68,9 @@
 #km-add-preset-btn {
   margin: 2px 0;
 }
+
+mat-card-content {
+  .mat-header-cell.km-header-cell.km-text-center {
+    text-align: center;
```

Comment:
Did you tried to use fxFlex for that?

Reply:
remove as styles are not required anymore due to 
- Due to `left-alignment` of text/heading
- Shorten heading of coulmns

Reply:
@cschieder Updated screenshot is attached:

fixes as per our consensus 👍🏽 
https://github.com/kubermatic/dashboard/pull/4412#discussion_r855116716